### DISCUSSION TITLE
How to make use of relative URLs in html snippets?

### DISCUSSION THREAD

**Discussion by ssbarnea** [squidfunk/mkdocs-material#6717] [Q&A] [answered]
I am using the `<video>` tag to include some videos into my documentation but I am unable to make them work for both cases: local build (mkdocs server) and deployed version (via RTD)

```md
# My markdown file

<video width="100%" controls autoplay loop>
<source src="../media/video.mp4" type="video/mp4">
</video>
```

To make it work on local build I need to preppend an extra `../` to the src but doing this will break the deployed version.

How can I make mkdocs build documentation with URLs that load the video for both cases?

**Comment by alexvoss** [comment, marked as answer]
Hi, using relative paths should really work. It is absolute paths that usually cause problems. Where do the videos live? Assets are normally within the `docs` directory so they get copies to `site`. Not sure why the path would be different on the server. What you say implies that specifying `media/video.mp4` works on the server?

**Reply by ssbarnea** [reply]
False alarm, the issue was caused by use of git-lfs which is not enabled on readthedocs. When not installed these media files endup with some bogus content instead of their original content.

I spend few good hours strying to make RTD to work with git-lfs before losing hope.

**Reply by alexvoss** [reply]
Ah, thanks for letting us know. I was wondering about what might be wrong. Computers are fun!
